code/app/src/main/java/com/example/bread/repository/ParticipantRepository.java

```diff
@@ -217,12 +220,14 @@ public void addFollower(@NonNull String username, String followerUsername, @NonN
         follower.put("username", followerUsername);
         getParticipantCollRef().document(username).collection("followers").document(followerUsername).set(follower)
                 .addOnSuccessListener(aVoid -> {
-                    // Update the follower count after adding
-                    fetchFollowers(username, followers -> {
-                        onSuccessListener.onSuccess(aVoid);
-                    }, e -> onSuccessListener.onSuccess(aVoid));
+                    // Directly increment the follower count by 1
+                    getParticipantCollRef().document(username).update("followerCount",
+                            com.google.firebase.firestore.FieldValue.increment(1));
```

Comment:
does it needs to be qualified? I prefer if we can move `com.google.firebase.firestore.FieldValue` in the imports